lighthouse-core/gather/gather-runner.js

```diff
@@ -91,7 +91,13 @@ class GatherRunner {
       }));
   }
 
-  static setupDriver(driver, options) {
+  /**
+   * @param {!Driver} driver
+   * @param {!Object} gathererResults
```

Comment:
`@param {!Object<string, !Array<!Promise<*>>} gathererResults` (should probably make a typedef at some point :)